catalog/transformer/catalog-transformer-zip/src/main/java/org/codice/ddf/catalog/transformer/zip/ZipCompression.java

```diff
@@ -103,7 +104,7 @@ public BinaryContent transform(SourceResponse upstreamResponse,
                 .forEach(metacard -> {
                     ZipParameters zipParameters = new ZipParameters();
                     zipParameters.setSourceExternalStream(true);
-                    zipParameters.setFileNameInZip(metacard.getId());
+                    zipParameters.setFileNameInZip(METACARD_PATH + metacard.getId());
```

Comment:
This `setFileNameInZip` method takes a String parameter and doesn't deal with the File object